README.md

```diff
@@ -34,7 +34,7 @@ the Grafana dashboards, some things may not show up.
 
 ## TL;DR
 
-To quickly get up and running, make sure you have the all [prerequisites](#Prerequisites) installed and then:
+To quickly get up and running, **make sure you have all the prerequisites installed** and then:
```

Comment:
I guess there is no point of making a link if it's 15 lines above

Reply:
Exactly, the requirements are just before this section.